ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -108,12 +115,16 @@ type ssh_client_options_type = {
 
 type ssh_daemon_type = {
     "options" ? ssh_daemon_options_type
+    "multiline_options" ? ssh_daemon_multiline_options_type
```

Comment:
i'm not a fan of refelcting the format in this way, `HostKey` should be configured under options like the other options. we typically use a static list of keys that should be converted in certain format. 
(and i liked my previous proposla of making `HostKey` a dict with valid key types :smile: )